Before: 9d6de05e909457c4714a14c22d8ae550c92ade09e7ceb69331556b282e636d87
After: 59c3b6a356b8fec691e6f168836a0fbb4676c607b5dde6b850df5744edb16d18
Patch: 1e6f0a6b732428afd9211d2d689094e4d679f0640b3ab02498fabd18e465e5e9
--- .gitignore	2012-07-22 20:44:49.485582560 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,17 +0,0 @@
-~*
-.~*
-*.bak
-*.log
-*.tmp
-*.out
-*.err
-*.pid
-.*.swp
-.*.swo
-.*.swn
-
-.settings
-.metadata
-.idea
-target
-sloccount.sc
--- Gemfile	2012-07-22 20:44:49.485582560 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,5 +0,0 @@
-source :rubygems
-
-# Load Gemfile.raven
-eval File.read(File.expand_path('../Gemfile.raven',__FILE__))
-
--- Gemfile.raven	2012-07-22 20:44:49.485582560 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,38 +0,0 @@
-# vim: ft=ruby et ts=2 sw=2 :
-header "# This file is generated by raven update:deps"
-footer "# v\im: ft=ruby et ts=2 sw=2 :"
-
-if options[:use_raven_gem]
-  gem 'raven', ">= #{::Raven::VERSION}"
-else
-  gem 'raven', path: RAVEN_ROOT
-end
-gem 'rake', group: :rake
-
-group :test do
-  gem 'simplecov'
-end if Dir.exists?('test')
-
-group :test_unit do
-  gem 'minitest'
-  gem 'guard-minitest', '>= 0.5.0'
-end if Dir.exists?('test/unit')
-
-group :test_spec do
-  gem 'rspec'
-  gem 'guard-rspec'
-end if Dir.exists?('test/spec')
-
-group :ci do
-  gem 'ci_reporter', require: false
-end
-
-group :doc do
-  gem 'yard'
-  gem 'rdiscount' # TODO aaaaaaaaaahhhhhhhhhh
-end
-
-group :debug do
-  gem 'debugger', platforms: :mri
-end
-
